cmd/chisel/main.go

```diff
@@ -25,6 +25,7 @@ var (
 	Stdout io.Writer = os.Stdout
 	Stderr io.Writer = os.Stderr
 	// overridden for testing
+	OpenArchive  = archive.Open
```

Comment:
> Based on the latter I wonder if instead we should make `OpenArchive` private and use `export_test` to change it.

I don't think we can do that. If we do here:
```
openArchive = archive.Open
```
Then we will call `openArchive` in cmd_cut.go, not `OpenArchive`.

And in `export_test.go`, then we do this:
```
OpenArchive = openArchive
```
We can't really change the archive usage or anything because we can't replace `openArchive` which is the one that will be used in cmd_cut.go.

Reply:
I don't think that will work. First `openArchive` being a function and even if that is supported if we assign `fakeOpenArchive` to `openArchive` in the `main` package (not `main_test`), then the actual functionalities will never be used.

Reply:
Alright, I finally understand what you meant from your texts. Changed in https://github.com/letFunny/chisel/pull/11/commits/993991f6fd2758ddca3c22078a69068b078dd2a4.

Reply:
it was not straightforward :D, I only knew because I found it in the code